code/workspaces/infrastructure-api/src/kubernetes/deploymentApi.spec.js

```diff
@@ -59,7 +60,7 @@ describe('Kubernetes Deployment API', () => {
     it('should return an error if creation fails', () => {
       mock.onPost(DEPLOYMENT_URL).reply(400, { message: 'error-message' });
 
-      return deploymentApi.createDeployment(DEPLOYMENT_NAME, manifest)
+      return deploymentApi.createDeployment(DEPLOYMENT_NAME, NAMESPACE, manifest)
         .catch((error) => {
           expect(error.toString()).toEqual('Error: Unable to create kubernetes deployment error-message');
         });
```

Comment:
It's been mentioned on other PRs but if `createDeployment` doesn't throw then I think this would still be a test pass. There are a few such tests in this file.